borg/helpers.py

```diff
@@ -457,19 +457,26 @@ def x(v):
     return '%s%s%s' % (x(mod // 64), x(mod // 8), x(mod))
 
 
-def format_file_size(v):
+def format_file_size(v, precision=2):
     """Format file size into a human friendly format
     """
-    if abs(v) > 10**12:
-        return '%.2f TB' % (v / 10**12)
-    elif abs(v) > 10**9:
-        return '%.2f GB' % (v / 10**9)
-    elif abs(v) > 10**6:
-        return '%.2f MB' % (v / 10**6)
-    elif abs(v) > 10**3:
-        return '%.2f kB' % (v / 10**3)
-    else:
-        return '%d B' % v
+    return sizeof_fmt_decimal(v, suffix='B', sep=' ', precision=precision)
```

Comment:
not sure i follow that either... you mean `format_file_size` should be able to return iec or decimal units? sounds illogical to me: we want files in decimal and memory in iec. the only reason why i kept `format_file_size` too is to avoid rewriting all the callers...